content/post/My-journey-with-ALiAS.md

```diff
@@ -0,0 +1,37 @@
++++
+title = "My journey with ALiAS : From a commit to the community"
+slug = "FOSS communities"
+image = "/images/community.jpg"
+date = "2019-05-25T00:00:00"
+tags='ALiAS, FOSS'
+categories = 'Shared experience'
```

Comment:
```suggestion
categories = 'Alumni journals'
```
Since, we want to keep all Alumni articles in one category.

Reply:
more like
```suggestion
categories = ['Shared experience']
```